packages/web/src/models/pool/mapper/pool-mapper.ts

```diff
@@ -38,15 +38,15 @@ export class PoolMapper {
       tokenA,
       tokenB,
       feeTier: feeTierInfo?.type || "NONE",
-      apr: apr !== null ? apr.toString() : "",
```

Comment:
It's probably a good idea to revert the changes on this line.

Reply:
it's some issue in fake data, i cheked it